README.md

```diff
@@ -29,3 +29,8 @@ What is the asymptotic runtime ($\Theta$) of your algorithm in terms of the
 number of comparisons? What is it in terms of the number of flips? Add your
 answer to this markdown file.
 
+The asymptotic runtime of pancake sort in terms of the number of comparisons is $\Theta(n^2)$.  This is true since the main operation for comparisons is the for loop that finds the greatest element in the unsorted part of the array, which has a runtime of $\Theta(n^2)$.  The runtime for the number of flips is $\Theta(n)$ since that is the runtime for the flip function that is called in the pancakeSort function.
+
+I worked on this assignment with Cole (Nathanael), Ashlyn, and Megan.  I also recieved help from ChatGPT.  ChatGPT told me I could remove a redundant while loop, and it also guided me to the answers for the runtime analysis.  It also gave me the idea of writing code that moves the greatest unsorted element to the beginning of the array using the flip function and then using flip again to move it to its correct position.
```

Comment:
It sounds like ChatGPT did most of the work here. Can you say explicitly what you did please?